apps/meteor/server/ufs/ufs-local.ts

```diff
@@ -69,11 +69,14 @@ export class LocalStore extends Store {
 		this.delete = async (fileId) => {
 			const path = await this.getFilePath(fileId);
 
-			const statResult = await stat(path);
-
-			if (statResult?.isFile()) {
-				await unlink(path);
-				await this.removeById(fileId);
+			try {
+				const statResult = await stat(path);
+				if (statResult?.isFile()) {
+					await unlink(path);
+					await this.removeById(fileId);
+				}
+			} catch (_e) {
+				// noop
```

Comment:
Is it good enough to silently fail? Wouldn't be better to update the callers of `this.delete` to do something if an error happens? Cause this way, there's no current indication of the file being unexistent, not even a log 👀